webpack/es5.config.js

```diff
@@ -20,8 +22,8 @@ module.exports = (env, argv) => {
       filename: `[name]${lightFilenamePart}.js`,
       chunkFilename: `[id]-[chunkhash]${lightFilenamePart}.js`
     },
-    optimization: optimization(argv.mode),
-    plugins: plugins(argv.mode)
+    optimization: optimization(mode),
```

Comment:
the 'optimization' function never returned anything.

Reply:
It returns the configuration only if the mode is production.